packages/blocks/src/api/factory.js

```diff
@@ -461,3 +462,17 @@ export function switchToBlockType( blocks, name ) {
 		return applyFilters( 'blocks.switchToBlockType.transformedBlock', transformedBlock, blocks );
 	} );
 }
+
+/**
+ * Create a block object from the example API.
+ *
+ * @param {string} name
+ * @param {Object} example
+ *
+ * @return {Object} block.
+ */
+export const getBlockFromExample = ( name, example ) => {
```

Comment:
Should it be `createBlockFromExample` for conistency with the default name and the description?
```suggestion
export const createBlockFromExample = ( name, example ) => {
```